content/blog/cloudnative-pg-1-19-1-released/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "CloudNativePG 1.19.1, 1.18.3 and 1.17.5 Released!"
+date: 2022-12-21T17:58:34+02:00
```

Comment:
does this stale date come from the hugo template? Or is it a copy-paste?

Reply:
latter :)